resources/api/reservation.py

```diff
@@ -479,6 +479,18 @@ def filter_reserver_info_search(self, queryset, name, value):
         # restrict results to reservations the user has right to see
         queryset = queryset.extra_fields_visible(user)
 
+        # assume that first_name and last_name were provided if empty space was found
+        if ' ' in value:
```

Comment:
These should be integrated to the `conditions` array below. As it stands having a search term with a space skips all the other fields, so a search `+358 50 123456` would not produce any results.

Reply:
Done!